Health check constants (#60)
Adamant-im/adamant-wallets: README.md

  // Optional. Node links for API
  "nodes": {
    "list": [
      { "url": "https://node.example.com" },
      { "url": "http://0.0.0.0:36666" }, // It's possible to use IP:port URI
      {
        "url": "https://second-node.example.com",
        "alt_ip": "0.0.0.1:36666" // Alternative way to connect if the domain of a node is censored
      }
    ],
    // Node additional information 
    "healthCheck": {
      "normalUpdateInterval": 210000, // Regular node status update interval in ms
      "crucialUpdateInterval": 30000, // Node status update interval when there are no active nodes, in ms
      "onScreenUpdateInterval": 10000, // On the node screen, the status update interval in ms
      "threshold": 3  // Permissible height difference between nodes
    },
    "minVersion": "1.0.0" // Optional. Minimal supported service API version      
  },

  // Optional. Services related to a project
  "services": {
    "list": {
      "infoService": [
        { "url": "https://info.example.com" },
        {
          "url": "https://second-service.example.com",
          "alt_ip": "0.0.0.1:80" // Alternative way to connect if the domain of a service is censored
        }
      ]
    },

>     // Optional: Service additional information (If not filled here, information is retrieved from nodes.additionalInfo)

Reviewer: Update the comment, nodes.additionalInfo is no more relevant

    "healthCheck": {

>       "normalUpdateInterval": 210000, // Optional. Regular service status update interval in ms
>       "crucialUpdateInterval": 30000, // Optional. Service status update interval when there are no active services, in ms
>       "onScreenUpdateInterval": 10000 // Optional. On the node screen, the status update interval in ms

Reviewer 2: ```suggestion
      "normalUpdateInterval": 210000, // Regular service status update interval in ms
      "crucialUpdateInterval": 30000, // Service status update interval when there are no active services, in ms
      "onScreenUpdateInterval": 10000 // On the node screen, the status update interval in ms
```

  "blockTimeFixed": 5000, // Optional. Fixed block time in ms
  "blockTimeAvg": 600000, // Optional. Average block time in ms

  // Node additional information 
  "nodeAdditionalInfo": {

>       "normalUpdateInterval": 210000, // Node status update interval in ms

Reviewer 2: ```suggestion
      "normalUpdateInterval": 210000, // Regular node status update interval in ms
```

>       "crucialUpdateInterval": 30000, // Crucial node status update interval in ms

Reviewer 2: ```suggestion
      "crucialUpdateInterval": 30000, // Node status update interval when there are no active nodes, in ms
```

      "onScreenUpdateInterval": 10000, // On the node screen, the status update interval in ms
      "threshold": 3,  // Permissible height difference between nodes

>       "normalServiceUpdateInterval": 210000, // Optional. Service node status update interval in ms

Reviewer 2: ```suggestion
      "normalServiceUpdateInterval": 210000, // Optional. Regular service status update interval in ms
```

>       "crucialServiceUpdateInterval": 30000, // Optional. Crucial service node status update interval in ms

Reviewer 2: ```suggestion
      "crucialServiceUpdateInterval": 30000, // Optional. Service status update interval when there are no active services, in ms
```

      "onScreenServiceUpdateInterval": 10000, // Optional. On the node screen, the status update interval in ms

>       "minNodeVersion": "0.8.0", // Optional. Minimal supported node API version

Reviewer 2: ```suggestion
      "minNodeVersion": "0.8.0", // Optional. Minimal supported node API version
      "minServiceVersion": "1.0.0", // Optional. Minimal supported service API version      
```
